content/docs/storage/platforms/powerflex/Product_Guide.md

```diff
@@ -291,20 +291,20 @@ Managing volumes on PowerFlex storage system includes creating new volume, mappi
 ### Examples
 <pre>
     <code>
-        resource "powerflex_volume" "avengers-volume-create"{
-        name = "avengers-volume-create"
-        protection_domain_name = "domain1"
-        storage_pool_name = "pool1" 
-        size = 8
-        use_rm_cache = true 
-        volume_type = "ThickProvisioned" 
-        access_mode = "ReadWrite"
+        resource "powerflex_volume" "volume-create"{
+        name = "[volume-name]"
```

Comment:
I'll suggest , better keep some values. Because for other resources and datasources, we'have kept some values.

Reply:
Just to be in sync